### PR TITLE
Refactoring accounts microservice with rpc standards 

### PR DESCRIPTION
**Pull request by vishnusudarshan**
# Description

This PR is for refactoring accounts microservice APIs with RPC standards from REST. This will have changes in the structure of the endpoint URLs in accounts microservice.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: accounts/src/routes/accounts.js [hunk 1]
```diff
@@ -1,8 +1,46 @@
 import express from 'express';
-import { unlockAccount } from '../services/accounts';
+import {
+  newAccount,
+  sendEtherToAccount,
+  getBalance,
+  pay,
```
Comment (ChaitanyaKonda) [inline accounts/src/routes/accounts.js]: Can we name this as transferEther instead of pay

Comment (iAmMichaelConnor) [reply accounts/src/routes/accounts.js]: What's the difference between transferEther and transferEtherToAccount? Do we need both?

Comment (vishnusudarshan) [reply accounts/src/routes/accounts.js]: @iAmMichaelConnor We can rename the naming from `sendEtherToAccount` to `transferEtherToAccount`. 
Then the functions :
`transferEtherToAccount` - Will transfer predefined value and gas to and account address from coinbase.
Whereas
`transferEther` - Will transfer a user-defined amount of value to the user.

Comment (vishnusudarshan) [reply accounts/src/routes/accounts.js]: Updated two functions to a single function with multi-purpose based on parameters.

**Pull request merged**